aspnet/getting-started/installing-on-linux.rst

```diff
@@ -100,10 +100,63 @@ You should now be able to restore packages from both the official public feed on
 
 
 Installing on CentOS, Fedora and derivatives
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+--------------------------------------------
 
-.. note::
+ The follwoing instrcution were tested using CentOS 7.
+ 
+Install Mono
```

Comment:
Instead of inlining a copy of the instructions for installing Mono let's just link to http://www.mono-project.com/docs/getting-started/install/linux/#centos-fedora-and-derivatives